openmetadata-docs/content/v1.3.x/deployment/upgrade/kubernetes.md

```diff
@@ -187,9 +187,9 @@ This issue is related to a minor change that affected the MySQL Database Engine
 
 As a result of the above fixes, anyone who is on OpenMetadata Dependencies Helm Chart Version `0.0.49` and `0.0.50` is affected with the above issue when upgrading for mysql. In order to fix this issue, make sure to follow the below steps -
 
-1. Backup the Database using Metadata Backup CLI as mentioned [here](#backup-your-data)
+1. Backup the Database using Metadata Backup CLI as mentioned [here](/deployment/upgrade/kubernetes#backup-your-metadata)
```

Comment:
You can just use `#backup-your-metadata` here

Reply:
This way we don't have to worry about this failing again, incase we change the path of this file